Helpers.gs

```diff
@@ -831,28 +831,36 @@ function sendSummary() {
  * @param {Number} maxRetries - How many times the function should try if it fails
  * @return {?Calendar.Event} The Calendar.Event that was added in the calendar, null if func did not complete successfully
  */
+var backoffRecoverableErrors = [
+  "Service invoked too many times in a short time",
+  "Rate Limit Exceeded",
+  "Internal error"];
 function callWithBackoff(func, maxRetries) {
   var tries = 0;
-  var timeout = 100;
   var result;
-  while(tries < maxRetries){
+  while ( tries < maxRetries ) {
```

Comment:
This has been wrong before but i think this should be _<=_. With maxRetries set to 0 i would expect the _func_ to get called once but not again if it fails. As of now it would not be called at all.

Reply:
Agreed. Change committed